ocf_data_sampler/select/geospatial.py

```diff
@@ -55,6 +55,23 @@ def lon_lat_to_osgb(
     return _lon_lat_to_osgb(xx=x, yy=y)
 
 
+def lat_lon_to_geostationary_area_coords(
```

Comment:
```suggestion
def lon_lat_to_geostationary_area_coords(
```

Very nitpicky but for the sake of consistency, that's what all other functions have